lib/reporters/mini.js

```diff
@@ -70,19 +70,16 @@ MiniReporter.prototype.prefix = function (str) {
 };
 
 MiniReporter.prototype._test = function (test) {
-	var title;
+	var title = test.title;
 
 	if (test.todo) {
-		title = colors.todo('- ' + test.title);
```

Comment:
> That's what this PR does.

Oh my God, what an embarrassment, I got confused with the new "Files" view and read the wrong column in diff...

> How about we just not show them all together? So if a slow test is running, the last actual test is shown, not todo/skip.

Even better.